pkg/portal/regions.go

```diff
@@ -0,0 +1,98 @@
+package portal
+
+// Copyright (c) Microsoft Corporation.
+// Licensed under the Apache License 2.0.
+
+import (
+	"encoding/json"
+	"fmt"
+	"net/http"
+	"os"
+
+	"github.com/Azure/ARO-RP/pkg/util/azureclient"
+)
+
+var (
+	PROD_REGIONS = []string{
+		"australiacentral",
+		"australiacentral2",
+		"australiaeast",
+		"australiasoutheast",
+		"brazilsouth",
+		"brazilsoutheast",
+		"canadacentral",
+		"canadaeast",
+		"centralindia",
+		"centralus",
+		"centraluseuap",
+		"eastasia",
+		"eastus",
+		"eastus2",
+		"eastus2euap",
+		"francecentral",
+		"germanywestcentral",
+		"japaneast",
+		"japanwest",
+		"koreacentral",
+		"northcentralus",
+		"northeurope",
+		"norwaywest",
+		"norwayeast",
+		"southafricanorth",
+		"southcentralus",
+		"southeastasia",
+		"southindia",
+		"swedencentral",
+		"switzerlandnorth",
+		"switzerlandwest",
+		"uaenorth",
+		"uaecentral",
+		"uksouth",
+		"ukwest",
+		"westcentralus",
+		"westeurope",
+		"westus",
+		"westus2",
+		"westus3",
+	}
+)
+
+type Region struct {
+	Name string `json:"name"`
+	URL  string `json:"url"`
+}
+
+type RegionInfo struct {
+	Regions []Region `json:"regions"`
+}
+
+func (p *portal) regions(w http.ResponseWriter, r *http.Request) {
+	resp := &RegionInfo{
+		Regions: make([]Region, 0, len(PROD_REGIONS)),
+	}
+
+	if value, found := os.LookupEnv("AZURE_ENVIRONMENT"); found {
+		var err error
```

Comment:
This may be some Go idiom I'm unfamiliar with but won't `err` be initialized as `nil` always, making the check in the next line useless?

Reply:
Nope, just a silly add on my side. Removed